salt/modules/systemd.py

```diff
@@ -158,6 +160,123 @@ def systemctl_reload():
     return retcode == 0
 
 
+def _default_runlevel():
```

Comment:
In the current form, I can think of a few ways that `runlevel` might be unset in the return statement. All of which would mean something was horribly broken, of course, but still... Would we maybe want to define a default runlevel as a parameter? i.e. `def _default_runlevel(runlevel=2):`. Or wrap the return with `if runlevel` and raise an exception if it's unset with a nice error message explaining that their system is broken? ;)